### PR TITLE
[Amazon AWS] Improvements for SQS, DynamoDB and CloudFormation commands

### PR DESCRIPTION
**Pull request by jfkisafk**
## Description

- Adds sort descending and consistent reads to DDB queries
- Saves the query projection expression in between form renders
- Improved toast error handling for SQS, DDB and CFN command actions
- Sorts the resources listed for SQS, DDB, CFN commands

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/amazon-aws/src/components/dynamodb/query-form.tsx [hunk 1]
```diff
@@ -15,10 +27,16 @@ interface QueryFormValues {
...
   projectionExpression: string;
+  isDescending: boolean;
 }
 
 export const QueryForm = ({ table }: { table: Table }) => {
-  const { push } = useNavigation();
+  const [lastProjectionExpression, setLastProjectionExpression] = useCachedState<string>(
+    "last-projection-expression",
+    "",
+    { cacheNamespace: table.TableName },
```
Comment (srikirank) [inline extensions/amazon-aws/src/components/dynamodb/query-form.tsx:37]: Could we add AWS account ID too?

**Review by srikirank — APPROVED** [review]

File: extensions/amazon-aws/src/components/dynamodb/query-form.tsx (hunk 1 above)
Comment (jfkisafk) [reply extensions/amazon-aws/src/components/dynamodb/query-form.tsx:37]: Oh yea good call regarding disambiguation, as two independent accounts might have the same table name. Although, the same table name might be intentional if cellular architecture is being used, for example, my-acct-beta and my-acct-prod might have the same table and can use the same projection for query. I think even in this case, it is better to let the use setup projection separately.

Comment (jfkisafk) [reply extensions/amazon-aws/src/components/dynamodb/query-form.tsx:37]: Just using TableArn for cacheNamespace in the followup commit

**Review by srikirank — APPROVED** [review]

**Review by JonathanWbn — APPROVED** [review]
😍

**Comment by jfkisafk** [comment]
@pernielsentikaer when you find some time please assist in merging the changes as they have been approved

**Comment by pernielsentikaer** [comment]
✅ Approved

**Comment by jfkisafk** [comment]
@pernielsentikaer sorry I had protection rules on my branches, can you please retry merging?

**Pull request merged**